docs/resources/tozny_pam_jira_plugin.md

```diff
@@ -0,0 +1,139 @@
+# tozny_pam_jira_plugin Resource
+
+A PAM (Privileged Access Management) Jira Plugin is a resource containing credentials required to
+integrate TozID with Jira. The plugins are used when configuring a Multi-party control access policy
+for a group that allows a Jira board to create Access Requests and manage their approval/denial.
+
+## Example Usage
+
+```hcl
+###
+# Prerequisites & Setup
+#
+# A PAM Jira Plugin requires an MPC-enabled realm. This section sets up the runtime & tozny provider
+# and creates a realm we can work with.
+###
+
+# Configure the Terraform runtime
+terraform {
+  required_version = ">= 0.14"
+  required_providers {
+    tozny = {
+      # Pull signed provider binaries from the Terraform hosted registry namespace
+      # for Tozny registry.terraform.io/tozny
+      source = "tozny/tozny"
+      # Pin Tozny provider version
+      version = ">=0.14.0"
+    }
+  }
+}
+
+# Include the Tozny Terraform provider
+provider "tozny" {
+  api_endpoint     = "http://platform.local.tozny.com:8000"
+  account_username = "test+${random_string.account_username_salt.result}@tozny.com"
```

Comment:
this should follow the pattern in the other documentation (now updated to use the test-emails-group prefix)

Reply:
ah, my bad. those other documentation changes didn't exist when i copied this code 😅 will update